specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -3983,6 +4231,121 @@
       },
       "description": "Filters limit replication to a subset of blobs within the storage account. A logical OR is performed on values in the filter. If multiple filters are defined, a logical AND is performed on all filters."
     },
+    "LocalUsers": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LocalUser"
+          },
+          "description": "The local users associated with the storage account."
+        }
+      },
+      "description": "List storage account local users."
+    },
+    "LocalUser": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/LocalUserProperties",
+          "x-ms-client-flatten": true,
+          "description": "Returns the storage account local user properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/Resource"
+        }
+      ],
+      "description": "The local user associated with the storage accounts."
+    },
+    "LocalUserProperties": {
```

Comment:
Which properties are readonly, and which can be set in create/update local user?
From example, seems only  permissionScopes can be set in create/update. If so, other properties should be readonly. Else please update example.